src/OpenTelemetry.Exporter.GoogleCloud/MetricsProviderBuilderExtensions.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+using OpenTelemetry.Internal;
+using OpenTelemetry.Metrics;
+using OpenTelemetry.Trace;
+
+namespace OpenTelemetry.Exporter.GoogleCloud;
+
+/// <summary>
+/// Extension methods to simplify registering a Stackdriver exporter.
+/// </summary>
+public static class MetricsProviderBuilderExtensions
```

Comment:
Could you please remove any new functionalities from rename PR?
I am not against introducing it, but it deserves own PR.